### PR TITLE
HS-971: Create spring based cucumber tests that live beside container cucumber tests

### PR DESCRIPTION
**Pull request by james8hut**
## Description
Create spring based cucumber tests that live beside container cucumber tests for easier development

## Jira link(s)
- https://issues.opennms.org/browse/HS-971

## Flagged for review
I was working on HS-941 (Add tenant id to alarms) before I realised it was sensible to split this story out. Some tenant id code from that story is in here, but it is all commented out. It will go in HS-941, which I will work on next.

Also, the test is a very basic one, to prove the piping works. It will be removed in HS-941

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

File: alarm/src/main/java/org/opennms/horizon/alarmservice/db/entity/TenantAwareEntity.java [hunk 1]
```diff
@@ -29,11 +29,17 @@
...
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.TenantId;
 
 /**
  * <p>Abstract Entity class.</p>
  */
+//@Getter
+//@Setter
+//@MappedSuperclass
```
Comment (thomasbigger584) [inline alarm/src/main/java/org/opennms/horizon/alarmservice/db/entity/TenantAwareEntity.java:42]: is this commented out code required?

File: alarm/src/main/java/org/opennms/horizon/alarmservice/db/tenant/TenantIdentifierResolver.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package org.opennms.horizon.alarmservice.db.tenant;
```
Comment (thomasbigger584) [inline alarm/src/main/java/org/opennms/horizon/alarmservice/db/tenant/TenantIdentifierResolver.java:1]: missing copyright headers - similar for a few other files

File: alarm/src/main/java/org/opennms/horizon/alarmservice/db/entity/TenantAwareEntity.java (hunk 1 above)
Comment (james8hut) [reply alarm/src/main/java/org/opennms/horizon/alarmservice/db/entity/TenantAwareEntity.java:42]: It's going back in later. See Flagged for review in PR.

File: alarm/src/main/java/org/opennms/horizon/alarmservice/db/tenant/TenantIdentifierResolver.java (hunk 2 above)
Comment (james8hut) [reply alarm/src/main/java/org/opennms/horizon/alarmservice/db/tenant/TenantIdentifierResolver.java:1]: will fix

**Review by thomasbigger584 — APPROVED** [review]

**Review by rabzobi — APPROVED** [review]
LGTM

**Pull request merged**